server/src/instant/admin/routes.clj

```diff
@@ -407,6 +409,16 @@
                         :headers {"app-id" (str counters-app-id)
                                   "authorization" (str "Bearer " admin-token)}}))
 
+;; Experimental. If we change this let the Kosmik folks know
+(defn schema-get [req]
+  (let [{app-id :app_id} (req->admin-token! req)
+        current-attrs (attr-model/get-by-app-id app-id)
+        current-schema (schema-model/attrs->schema current-attrs)]
+    (response/ok {:schema (update current-schema
+                                  :refs
+                                  update-keys
+                                  (partial string/join "-"))})))
```

Comment:
I do this, as the key on `refs` is a vec. I don't think we use this vec in any place, but instead of potentially introducing a bug, thought for now I'd update it here, so it becomes:

{
  "schema": {
    "refs": {
      "cats-people-people-cats": {
        "forward-identity": [